v22.1/ui-queues-dashboard.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Queues Dashboard
+summary: The Queues dashboard lets you monitor the health and performance of various queueing systems in CockroachDB.
+toc: true
+docs_area: reference.db_console
+---
+
+The **Queues** dashboard lets you monitor the health and performance of various queueing systems in CockroachDB.
+
+To view this dashboard, [access the DB Console](ui-overview.html#db-console-access) and click **Metrics** on the left-hand navigation, and then select **Dashboard** > **Queues**.
+
+{% include {{ page.version.version }}/ui/ui-metrics-navigation.md %}
+
+{{site.data.alerts.callout_info}}
+All timestamps in the DB Console are shown in Coordinated Universal Time (UTC).
+{{site.data.alerts.end}}
+
+The **Queues** dashboard displays the following time series graphs:
+
+## Queue Processing Failures
+
+<img src="{{ 'images/v22.1/ui_queue_failures.png' | relative_url }}" alt="DB Console queue failures graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Queue Processing Failures** graph shows you processing failures experienced across various queuing systems.
+
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+GC | The number of replicas which failed processing in the garbage collection queue, as tracked by the `queue.gc.process.failure` metric.
+Replica GC | The number of replicas which failed processing in the replica garbage collection queue, as tracked by the `queue.replicagc.process.failure` metric.
+Replication | The number of replicas which failed processing in the replica queue, as tracked by the `queue.replicate.process.failure` metric.
+Split | The number of replicas which failed processing in the split queue, as tracked by the `queue.split.process.failure` metric.
+Consistency | The number of replicas which failed processing in the consistency checker queue, as tracked by the `queue.consistency.process.failure` metric.
+Raft Log | The number of replicas which failed processing in the Raft log queue, as tracked by the `queue.raftlog.process.failure` metric.
+Raft Snapshot | The number of replicas which failed processing in the Raft repair queue, as tracked by the `queue.raftsnapshot.process.failure` metric.
+Time Series Maintenance | The number of replicas which failed processing in the time series maintenance queue, as tracked by the `queue.tsmaintenance.process.failure` metric.
+
+## Queue Processing Times
+
+<img src="{{ 'images/v22.1/ui_queue_time.png' | relative_url }}" alt="DB Console queue processing time graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Queue Processing Times** graph shows you the processing rate across various queue systems.
```

Comment:
I soft-suggest replacing "shows you" with just "shows" or "displays". It shows (or displays, which is less visible, as the display could technically be rendered by a screen-reader) it to anyone, not just 'you', even if 'you' refers to anyone reading. Not gonna repeat this suggestion everywhere as that would be super annoying in case you choose to ignore it lol